clients/venice-client/src/main/java/com/linkedin/venice/fastclient/StatsAvroGenericStoreClient.java

```diff
@@ -163,6 +167,13 @@ private <R> CompletableFuture<R> recordRequestMetrics(
          * considered an unhealthy request: so only incrementing a subset of metrics.
          */
         exceptionReceived = true;
+        if (!redundantExceptionFilter.isRedundantException(throwable.getMessage())) {
```

Comment:
I am not sure if this message content is consistent. Can we just use the store name, or does it support passing nothing and always print one log per minute?

Reply:
That is a good point. To start with, we can get all unique logs per minute and once validated and fixed, we can remove this or make it a stricter one log per minute.